package-lock.json

```diff
@@ -16954,6 +16954,11 @@
         "string-width": "^1.0.2 || 2"
       }
     },
+    "windows-iana": {
+      "version": "3.1.0",
+      "resolved": "https://registry.npmjs.org/windows-iana/-/windows-iana-3.1.0.tgz",
+      "integrity": "sha512-rCPf3AakAAgvapnbYVvG2bQyI3g6EDbPpjDJ72fdAu+XTzB1qvX4ZC6OnZ0I2+thaspjTb+8KwdyhdBl8Lt/QA=="
+    },
```

Comment:
What happened here? I guess package-lock wasn't updated before with this dep?

Reply:
I think it might be a new transitive dependency

Reply:
Yep, this was a PR I merged and suspect I missed that package-lock needed updating.

Reply:
That is ^ exactly what happened https://github.com/jitsi/jitsi-meet/pull/4327

Reply:
Ah, good to know, thanks @damencho